### ISSUE TITLE
My code stopped here for a long time, is this right？

### ISSUE THREAD

**Issue by ashencode** [jd730/OICR-pytorch#19]
My code stopped here for a long time,about 3 days， is this right？How much time does it take to train？
> /home/yesenmao/shenjt/OICR-pytorch/lib/model/oicr/oicr.py(37)WeightedSoftmaxWithLoss()
-> loss = (labels_ic * torch.log(prob + eps))
(Pdb) > /home/yesenmao/shenjt/OICR-pytorch/lib/model/oicr/oicr.py(37)WeightedSoftmaxWithLoss()
-> loss = (labels_ic * torch.log(prob + eps))
(Pdb)

**Comment by jd730** [comment]
Hi @ashencode , I am sorry that one `pdb.set_trace()` is added on https://github.com/jd730/OICR-pytorch/blob/97c8ecccfb186f35acd89f4537e5f45a245ed817/lib/model/oicr/oicr.py#L36 at 6ec17f66e65d971af0b9c5f1655aa6cfbbc99d35.
Please delete that line and it will run correctly.
